### ISSUE TITLE
Server files for 0.5B seem to be incomplete

### ISSUE THREAD

**Issue by natebc** [LiviiYS/Euphoric_Curiosity#22]
```
Caused by: net.fabricmc.loader.impl.discovery.ModResolutionException: Mod resolution encountered an incompatible mod set!                                                                     A potential solution has been determined:                                                                                                                                                              - Replace mod 'Modern Industrialization' (modern_industrialization) 1.0.0 (/var/games/minecraft/servers/euphoric/mods/Modern-Industrialization-1.0.0.jar) with any version that is compatible with:                                                                                                                                                                                Unmet dependency listing:                                                                                                                                                                              - Mod 'Modern Industrialization' (modern_industrialization) 1.0.0 requires any version of roughlyenoughitems, which is missing! 
```
Fresh install of 0.5B.
This is also mentioned at the tail of the comments in #17 (closed).

Please let me know if any further information is needed.

**Comment by LiviiYS** [comment]
Fixed on 0.6

**Comment by natebc** [comment]
thanks for fixing this so quickly!

I checked on curseforge and I think they may be slow to approving your server files release for 0.6. I see the client manifest release but no server files there yet.
